### PR TITLE
Update EDPM timeout

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

File: devsetup/README.md [hunk 1]
```diff
@@ -84,52 +84,7 @@ make download_tools
...
-make edpm_play_cleanup
-# Will delete VM's!:
-make edpm_compute_cleanup
-```
-
-In case additional compute node VMs are deployed, run:
-```
-make edpm_compute_cleanup EDPM_COMPUTE_SUFFIX=1
-```
+See [Deploy dev env using CRC, edpm nodes with isolated networks](../README.md#deploy-dev-env-using-crc-edpm-nodes-with-isolated-networks)
```
Comment (slagle) [inline devsetup/README.md:87]: `../README.md` doesn't say anything about having to run `make edpm_compute_repos` using the devsetup Makefile.

**Review by slagle — CHANGES_REQUESTED** [review]

File: devsetup/README.md (hunk 1 above)
Comment (fao89) [reply devsetup/README.md:87]: because it is not needed anymore, as the repo-setup is service is being added

Comment (fao89) [reply devsetup/README.md]: I've removed this change, focus only on timeout for this PR

**Review by slagle — APPROVED** [review]

**Pull request merged**